src/DynamoCore/Graph/Workspaces/SerializationConverters.cs

```diff
@@ -534,43 +534,26 @@ public override object ReadJson(JsonReader reader, Type objectType, object exist
             var view = obj["View"];
             if (view != null)
             {
-                var nodesView = view["NodeViews"];
-                if (nodesView != null)
+                var nodeViews = view["NodeViews"].ToList();
```

Comment:
it's possible that NodeViews is null - this will throw.

Reply:
I actually found that anything broken in the Json will cause the file open to be interrupted, I can move up the try catch below to at least guard all the node view part